Realm/RealmSwift/Migration.swift

```diff
@@ -102,18 +102,58 @@ public func setSchemaVersion(schemaVersion: UInt, realmPath: String, migrationBl
 }
 
 /**
+Get the schema version for a Realm at a given path.
+
+:param: realmPath Path to a Realm file.
+:param: error     If an error occurs, upon return contains an `NSError` object
+                  that describes the problem. If you are not interested in
+                  possible errors, omit the argument, or pass in `nil`.
+
+:returns: The version of the Realm at `realmPath` or `nil` if the version cannot be read.
+*/
+public func schemaVersionAtPath(realmPath: String, error: NSErrorPointer = nil) -> UInt? {
```

Comment:
I'd prefer sticking to `UInt` to represent natural numbers, unless there's a strong parallel that can be made with a similar interface in the Swift standard library that does otherwise (e.g. `Array` subscript takes an `Int`).